text/0013-portable-pdb.md

```diff
@@ -0,0 +1,105 @@
+If that header is not provided, the server will always respond with a `403` error code. Otherwise, it uses a `404` code when the symbol is not found.
+
+Examples:
+
+```
+> curl https://symbols.nuget.org/download/symbols/microsoft.maui.pdb/10f7f174e11949f587c4d0b617742d31FFFFFFFF/microsoft.maui.pdb -i
+HTTP/1.1 403 Forbidden
+
+> curl https://symbols.nuget.org/download/symbols/microsoft.maui.pdb/10f7f174e11949f587c4d0b617742d31FFFFFFFF/microsoft.maui.pdb -H "SymbolChecksum: SHA256:74f1f71019e1f5197c4d0b617742d31b515f041c4f62a41bfc25ebb05a7fbff3" -i
+HTTP/1.1 404 Not Found
+```
+
+The details of this header do not seem to be publicly documented, but can be traced by looking through the `symstore` code:
+
+- PdbChecksum: https://github.com/dotnet/symstore/blob/aa44862e5028cb7595bbd474da1d63e8c24bf718/src/Microsoft.FileFormats/PE/PEStructures.cs#L375
+- PE Key: https://github.com/dotnet/symstore/blob/aa44862e5028cb7595bbd474da1d63e8c24bf718/src/Microsoft.SymbolStore/KeyGenerators/PEFileKeyGenerator.cs#L75
+- Portable PDB Key: https://github.com/dotnet/symstore/blob/aa44862e5028cb7595bbd474da1d63e8c24bf718/src/Microsoft.SymbolStore/KeyGenerators/PortablePDBFileKeyGenerator.cs#L85
+- Key Format: https://github.com/dotnet/symstore/blob/aa44862e5028cb7595bbd474da1d63e8c24bf718/src/Microsoft.SymbolStore/KeyGenerators/KeyGenerator.cs#L166-L177
+- `SymbolChecksum` Header: https://github.com/dotnet/symstore/blob/8a7c47ac74302510f839cc2361ca591b6f4df542/src/Microsoft.SymbolStore/SymbolStores/HttpSymbolStore.cs#L112
+
+# Options Considered
+
+## Stack Trace Interface Extensions
+
+We propose to add the following to the [Stack Trace Interface Frame Attributes](https://develop.sentry.dev/sdk/event-payloads/stacktrace/#frame-attributes):
+
+- `instruction_offset: Number`, as it is currently used by the `sentry-dotnet` SDK.
+- `method_index: Number`.
+
+The `instruction_offset` is defined as a JSON `Number`.
+This instruction offset is fetched using the .NET [`StackFrame.GetILOffset`](https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.stackframe.getiloffset)
+method which returns a `Int32` type.
+
+The `method_index` is a JSON `Number` for similar reasons.
+The method index is derived from the .NET [`MemberInfo.MetadataToken`](https://learn.microsoft.com/en-us/dotnet/api/system.reflection.memberinfo.metadatatoken)
+field which also has type `Int32`. This `MetadataToken` is also using a format that encodes the actual method index as the lower 3 bytes (24 bits).
+
+### Alternatives
+
+WASM stack frames can also refer to the concept of a function index. An alternative would be to use a property named `function_index` for both.
+
+## Debug Meta Interface Extensions
+
+We propose to add the following to the [Debug Images Interface](https://develop.sentry.dev/sdk/event-payloads/debugmeta#debug-images):
+
+- A new debug image type similar to the existing native image types.
+- It should have a `type: "portable-pe"` attribute.
```

Comment:
PE already stands for "Portable Executable". Such files are in fact regular PEs, though they contain special sections. I would ask to add reasoning on why a dedicated image type is required as opposed to just calling this `pe`. 

As a suggestion for naming, you could call this `pe_net` or `pe_assembly`, though I would defer to others for better suggestions. 

Finally, please consider snake_case rather than kebab-case. Outside of CSP, our protocol almost exclusively uses snake_case.